src/test/java/com/capitalone/dashboard/event/TestResultEventListenerTest.java

```diff
@@ -0,0 +1,175 @@
+package com.capitalone.dashboard.event;
+
+import com.capitalone.dashboard.model.TestResult;
+import com.capitalone.dashboard.model.TestCapability;
+import com.capitalone.dashboard.model.TestSuiteType;
+import com.capitalone.dashboard.model.TestCase;
+import com.capitalone.dashboard.model.TestCaseStatus;
+import com.capitalone.dashboard.model.TestCaseStep;
+import com.capitalone.dashboard.model.TestSuite;
+import com.capitalone.dashboard.model.Collector;
+import com.capitalone.dashboard.model.CollectorItem;
+import com.capitalone.dashboard.model.CollectorType;
+import com.capitalone.dashboard.model.Performance;
+import com.capitalone.dashboard.model.PerformanceType;
+import com.capitalone.dashboard.repository.CollectorItemRepository;
+import com.capitalone.dashboard.repository.CollectorRepository;
+import com.capitalone.dashboard.repository.PerformanceRepository;
+import org.bson.types.ObjectId;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.data.mongodb.core.mapping.event.AfterSaveEvent;
+
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TestResultEventListenerTest {
+
+    @Mock
+    private CollectorRepository collectorRepository;
+    @Mock
+    private CollectorItemRepository collectorItemRepository;
+    @Mock
+    private PerformanceRepository performanceRepository;
+    @InjectMocks
+    private TestResultEventListener testResultEventListener;
+    private static final String COLLECTOR_NAME = "PerfTools";
+
+
+    @Test
+    public void test_testResultListener(){
+        TestResult testResult = getTestResult(getTestResultCollectorItem());
+        CollectorItem perfCItem = getNewPerfCollectorItem(getPerfToolsCollector());
```

Comment:
rename perfCItem to perfCollItem to be consistent with the Class naming

Reply:
renamed